src/index.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { interpret } from 'xstate';
+import { SyncMachine } from './machine';
+import { Connection } from '@hathor/wallet-lib';
+
+const machine = interpret(SyncMachine).start();
+
+machine.onTransition(state => {
+  if (state.changed) {
+    console.log('Transitioned to state: ', state.value);
```

Comment:
Maybe we should use `winston` for logging? See an example here: https://github.com/HathorNetwork/hathor-wallet-headless/blob/master/src/logger.js

Reply:
I agree, done in: d13009045fbf876a0edbb2c7a9cc24abe6132a7d

The difference between my implementation and the Headless's is that I removed the config file entirely, making the log level the only configuration available (on an env variable).

I think it is better to leave the responsibility of managing logs to the deploy mechanism

Reply:
Ok. We can easily change it in the future.